### DISCUSSION TITLE
Full page examples

### DISCUSSION THREAD

**Discussion by Jamesking56** [mantinedev/mantine#297] [Q&A] [answered]
Hi,

I'm looking for some examples showing a marketing-style landing page using Mantine and an admin panel using Mantine.

Are there any examples out there for these?

Cheers.

**Comment by rtivital** [comment, marked as answer]
Hi, you can find all examples in documentation, that's all we have right now

**Reply by kvnwolf** [reply]
It could be a good idea to have something like Tailwind UI or the Premium Chakra UI components marketplace for monetization.

**Reply by qoobes** [reply]
Nice prediction hahaha
